src/graphql/client/graphql-client.ts

```diff
@@ -490,7 +490,7 @@ export class GraphQLClient {
       [`del: delete_${model}`]: {
         __args: {
           where: {
-            _and: [{id: {_in: ids}, ...deleteConditions}],
+            _and: [{...deleteConditions, id: {_in: ids}, }],
```

Comment:
remove comma - `[{...deleteConditions, id: {_in: ids} }]` ?

Reply:
Change this a little